packages/block-library/src/navigation/edit/navigation-menu-selector.js

```diff
@@ -173,27 +173,22 @@ function NavigationMenuSelector( {
 							} ) }
 						</MenuGroup>
 					) }
-
-					{ canUserCreateNavigationMenu && (
-						<MenuGroup label={ __( 'Tools' ) }>
-							<MenuItem
-								disabled={ isCreatingMenu }
-								onClick={ () => {
-									onClose();
-									onCreateNew();
-									setIsCreatingMenu( true );
-								} }
-							>
-								{ __( 'Create new menu' ) }
-							</MenuItem>
-						</MenuGroup>
-					) }
 				</>
 			) }
 		</DropdownMenu>
 	);
 
-	return NavigationMenuSelectorDropdown;
+	// TODO If there are more than one...
+	if (
+		( showNavigationMenus &&
+			hasNavigationMenus &&
+			navigationMenus?.length > 1 ) ||
+		( showClassicMenus && hasClassicMenus )
+	) {
+		return NavigationMenuSelectorDropdown;
+	}
+
+	return currentTitle ? currentTitle : __( 'Navigation' );
```

Comment:
```suggestion
	return currentTitle ? currentTitle : __( 'Navigation' );
```

We need to decode the entities. Not sure if it's here, but we should aim to do it as close to where the text is _rendered_ as possible.